components/add_new_cv_form.tsx

```diff
@@ -69,6 +69,20 @@ export default function AddNewCvForm({ id }: Props) {
     setForm(formData);
   }
 
+  async function uploadPdf(fileName: string) {
+    const response = await fetch(`/api/upload_to_drive?file_name=${fileName}`, {
+      method: "GET",
+    });
+    return response
```

Comment:
Can we check if the upload was successful in the response? And then return that value, which we can then use on [this line](https://github.com/bditengineering/cv-app/pull/18/files#diff-2cb1575ced49851863b2876bff6bd1c73a6fdd21908685e6695ce1bae306825cR184) to let the user know if the upload was successful and route to the homepage only if everything went fine.